packages/ckeditor5-dev-ci/lib/notifytravisstatus.js

```diff
@@ -34,19 +34,19 @@ module.exports = async function notifyTravisStatus( options ) {
 		notifyCommitUrl: options.notifyCommitUrl,
 		repositorySlug: options.repositorySlug
 	} );
-	const commitDetails = await getCommitDetails( commitUrl, options.githubToken );
+	const { githubAccount, commitAuthor, commitMessage } = await getCommitDetails( commitUrl, options.githubToken );
 
 	const [ buildRepoOwner, buildRepoName ] = options.repositorySlug.split( '/' );
-
-	const slackAccount = members[ commitDetails.author ];
+	const slackAccount = members[ githubAccount ] || null;
```

Comment:
Setting `null` here only to match the possible `slackAccount` values with the `githubAccount` values: both may be `string` or `null`.